kuksa_apps/kanto/config/edge_client.ini

```diff
@@ -0,0 +1,11 @@
+[SERVER]
+# host: default 127.0.0.1
+host = 192.168.56.51
+
+[SUBSCRIPTION]
+topic_dev_info = "edge/thing/response"
+topic_dev_info_request = "edge/thing/request"
+
+[]
+# signal paths
+path = "Vehicle.Cabin.Infotainment.Navigation.CurrentLocation"
```

Comment:
The bridge application currently processes only a specific subset of the CurrentLocation VSS tree values by default. It would be nice to have a collection of the exact paths here separated by `, ` - e.g. `Vehicle.Cabin.Infotainment.Navigation.CurrentLocation.Altitude, Vehicle.Cabin.Infotainment.Navigation.CurrentLocation.Latitude, Vehicle.Cabin.Infotainment.Navigation.CurrentLocation.Longitude, Vehicle.Cabin.Infotainment.Navigation.CurrentLocation.Speed`

As this property will actually hold a collection of values, it would be better to call it `paths` or `vss_paths`.

Reply:
The path used has been deprecated since VSS 2.1 and is removed in VSS 3.0. We will soon integrate VSS 3.0 (see #322) and make it default. I do not know (for this use case) if it is strictly needed to use a signal name that exists in the VSS *.json, or if the system can handle arbitrary names like `X.Y.Z`as well. (Do you know @SebastianSchildt ?)

Consider using `Vehicle.CurrentLocation` instead.

Reply:
@konstantina-gramatova 
as discussed I would go for VSS 3.0 paths, that is 
```
Test Client> getValue Vehicle/CurrentLocation/

PATH                                        Description                                                                                                               
Vehicle/CurrentLocation/Altitude            Sensor: Current altitude relative to WGS 84 reference ellipsoid, as measured at the position of GNSS receiver antenna.      
Vehicle/CurrentLocation/GNSSReceiver        Branch: Information on the GNSS receiver used for determining current location.                                             
Vehicle/CurrentLocation/GNSSReceiver/       Children of branch Vehicle/CurrentLocation/GNSSReceiver                                                                     
Vehicle/CurrentLocation/Heading             Sensor: Current heading relative to geographic north. 0 = North, 90 = East, 180 = South, 270 = West.                        
Vehicle/CurrentLocation/HorizontalAccuracy  Sensor: Accuracy of the latitude and longitude coordinates.                                                                 
Vehicle/CurrentLocation/Latitude            Sensor: Current latitude of vehicle in WGS 84 geodetic coordinates, as measured at the position of GNSS receiver antenna.   
Vehicle/CurrentLocation/Longitude           Sensor: Current longitude of vehicle in WGS 84 geodetic coordinates, as measured at the position of GNSS receiver antenna.  
Vehicle/CurrentLocation/Timestamp           Sensor: Timestamp from GNSS system for current location, formatted according to ISO 8601 with UTC time zone.                
Vehicle/CurrentLocation/VerticalAccuracy    Sensor: Accuracy of altitude.                     
```

I think they can be hardcoded (shoudl be relatively stable). Make sure you only read, what you set (e.g. current behaviour is, if you query `Vehicle/CurrentLocation/`  but one of the childs has never been set, you do not get any data). So to be also stable against further extensiosn of VSS for now the safe - albeit maybe not elegant -  way is readint the individual paths.

For testing with VSS 3.0, the current container in master has support built in, e.g.

`docker run -it --rm --net=host ghcr.io/eclipse/kuksa.val/kuksa-val:master`

gives you what you need (the release 0.2.1 of the pypi package is still cmpatible)

Reply:
Luckily, the Kanto bridge does not set any values (I guess the chosen feeder maybe does?) and for the sake of VSS path to Ditto prop path mapping, all the paths (configured in the .ini) are queried with a full path one by one.


So I guess, if the person creating the configuration is aware what values will be available, will be able to configure the paths to be queried properly. What do you think?

Reply:
I agree. Was too mich thinking about the demo I saw, and not that this is a general Kanto Adapter :D Yeah: It is a "deployment issue" and you can expect a person who is setting this up to be aware, what is available